GDM.HW3.ConditionalsMethods/Program.cs

```diff
@@ -0,0 +1,315 @@
+            {
+                return 29;
+            }
+            if (month == 2)
+            {
+                return 28;
+            }
+            if (month % 2 == 1 && month < 7 || month > 8 && month % 2 == 1)
+            {
+                return 30;
+            }
+            else
+            {
+                return 31;
+            }
+        }
+
+        //Task 5
+        static int[] ReccursiveQuickArraySort(int[] array, int firstArrayIndex, int lastArrayIndex)
+        {
+            int f = firstArrayIndex;
+            int l = lastArrayIndex;
+            int mid = array[(f + l) / 2];
+            do
+            {
+                while (array[f] < mid)
+                    ++f;
+                while (array[l] > mid)
+                    --l;
+                if (f <= l)
+                {
+                    if (f < l)
+                    {
+                        int temp = array[f];
+                        array[f] = array[l];
+                        array[l] = temp;
+                    }
+                    ++f;
+                    --l;
+                }
+            }
+            while (f <= l);
+            if (firstArrayIndex < l)
+            {
+                ReccursiveQuickArraySort(array, firstArrayIndex, l);
+            }
+            if (f < lastArrayIndex)
+            {
+                ReccursiveQuickArraySort(array, f, lastArrayIndex);
+            }
+            return array;
+        }
+
+        //Task 6
+        static int[] PutZeroBetweenMimMaxValueArray(int[] array)
+        {
+            int[] tempArray = (int[])array.Clone();
+            int[] sortedRandomArray = ReccursiveQuickArraySort(tempArray, 0, tempArray.Length - 1);
+            int min = 0;
+            int max = 0;
+
+            for (int i = 0; i < array.Length; i++)
+            {
+                if (array[i] == sortedRandomArray[0])
+                {
+                    min = i;
+                }
+            }
+            for (int i = 0; i < array.Length; i++)
+            {
+                if (array[i] == sortedRandomArray[sortedRandomArray.Length - 1])
+                {
+                    max = i;
+                }
+            }
+            if (min > max)
+            {
+                for (int i = max + 1; i < min; i++)
+                {
+                    array[i] = 0;
+                }
+            }
+            else
+            {
+                for (int i = min + 1; i < max; i++)
+                {
+                    array[i] = 0;
+                }
+            }
+            return array;
+        }
+
+        //Task 8
+        static bool IsDupl(int[] array, int veriable, int arrayIndex)
+        {
+            bool temp = false;
+            for (int i = 0; i < array.Length; i++)
+            {
+                if (array[i] == veriable && arrayIndex != i)
+                {
+                    temp = true;
+                    return temp;
```

Comment:
You don't need to **return temp** (you already have this on 138 line) here - just break for.

Reply:
oops. I know. Just fogot to delete it.